br/pkg/lightning/backend/local/localhelper.go

```diff
@@ -471,8 +466,8 @@ func (local *local) waitForScatterRegion(ctx context.Context, regionInfo *split.
 	}
 }
 
-func (local *local) isScatterRegionFinished(ctx context.Context, regionID uint64) (bool, error) {
-	resp, err := local.splitCli.GetOperator(ctx, regionID)
+func (local *local) isScatterRegionFinished(ctx context.Context, regionInfo *split.RegionInfo) (bool, error) {
```

Comment:
maybe rename this func to reflect it retry scatter inside

Reply:
Is it ok to rename it to `checkScatterRegionFinsihedOrReScatter`?